build.assets/images.mk

```diff
@@ -3,11 +3,12 @@
 # These values may need to be updated in `dronegen/container_image_products.go` if
 # they change here
 BUILDBOX_VERSION ?= teleport12
+BUILDBOX_BASE_NAME ?= public.ecr.aws/gravitational/teleport-buildbox
 
-BUILDBOX=public.ecr.aws/gravitational/teleport-buildbox:$(BUILDBOX_VERSION)
-BUILDBOX_FIPS=public.ecr.aws/gravitational/teleport-buildbox-fips:$(BUILDBOX_VERSION)
-BUILDBOX_CENTOS7=public.ecr.aws/gravitational/teleport-buildbox-centos7:$(BUILDBOX_VERSION)
-BUILDBOX_CENTOS7_FIPS=public.ecr.aws/gravitational/teleport-buildbox-centos7-fips:$(BUILDBOX_VERSION)
-BUILDBOX_ARM=public.ecr.aws/gravitational/teleport-buildbox-arm:$(BUILDBOX_VERSION)
-BUILDBOX_ARM_FIPS=public.ecr.aws/gravitational/teleport-buildbox-arm-fips:$(BUILDBOX_VERSION)
-BUILDBOX_TELETERM=public.ecr.aws/gravitational/teleport-buildbox-teleterm:$(BUILDBOX_VERSION)
+BUILDBOX=$(BUILDBOX_BASE_NAME):$(BUILDBOX_VERSION)
```

Comment:
So we won't use ECR at all anymore for the buildbox? Or will we still have both?

Reply:
Not during the CI. Drone still publishes buildboxes to ECR though and they are used in the Drone pipelines, but we might be able to get rid of them when we fully migrate to GHA.